### PR TITLE
[PS-181] update electron

### PR DESCRIPTION
**Pull request by jlf0dev**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Update electron to `19.0.8`
~~Update electron-builder to `23.1.0`~~ (new version still breaks the signing process)
Update electron-log to `4.4.8`
Update electron-store to `8.0.2`
Update electron-updater to `5.0.5`

There hasn't been any serious breaking changes in electron for these two versions:
https://www.electronjs.org/docs/latest/breaking-changes#planned-breaking-api-changes-190

## Before you submit

```
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

**Comment by Hinton** [comment]
We need to be careful when bumping electron builder and updater, builder didn't properly support our certs for testflight before. And updater needs a QA round to ensure the updates behave properly.

**Review by djsmith85 — DISMISSED** [review]
Changes look good, besides re-buiding `package-lock.json`.

For the electron-builder issue I have a created a follow-up task in our backlog.

File: package-lock.json [hunk 1]
```diff
@@ -186,7 +186,7 @@
     },
     "apps/cli": {
       "name": "@bitwarden/cli",
-      "version": "1.22.1",
+      "version": "2022.6.2",
```
[the remark this answers is not in the document]
Comment (jlf0dev) [reply package-lock.json:189]: I'm a little confused by this. We have the `2022.6.2` version in the cli project, so i'm assuming its copying it from there. Is there a reason it should stay as `1.22.1`?

[the remark this answers is not in the document]
Comment (djsmith85) [reply package-lock.json:189]: No, the bump to that version is fine, but it should not be showing up in the diff, as you did not make that change.

File: package.json [hunk 2]
```diff
@@ -82,14 +82,14 @@
     "cross-env": "^7.0.3",
     "css-loader": "^6.5.1",
     "del": "^6.0.0",
-    "electron": "16.2.7",
-    "electron-builder": "22.11.10",
-    "electron-log": "^4.4.7",
+    "electron": "19.0.8",
+    "electron-builder": "^22.11.10",
```
Comment (MGibson1) [inline package.json:86]: Please keep this at this exact version

**Review by MGibson1 — CHANGES_REQUESTED** [review]

**Comment by MGibson1** [comment]
@vgrassia can you please retest testflight with the latest changes?

File: package-lock.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (djsmith85) [reply package-lock.json:189]: Spoke about this earlier today and it seems that the cli version bump action only bumped the version in the `apps/cli/package.json` and `apps/cli/package-lock.json` and did not commit the change of the root `package.json` and `package-lock.json`.

Error in the build action: Not committing those changes properly. 
@jlf0dev just being the first dev to rebuild the package-lock.json and that's why the change shows up in the diff

**Review by MGibson1 — APPROVED** [review]

**Pull request merged**